pyslackers_website/slack/urls.py

```diff
@@ -1,11 +1,12 @@
 from django.urls import path
 
-from .views import SlackInvite, timezone_json_view
+from .views import SlackInvite, timezone_json_view, monthlymemberships_json_view
```

Comment:
This spreads across a few locations, but `monthlymemberships` would be more readable as `monthly_memberships` given they are 2 words